### PR TITLE
Add {{compute}} helper

### PR DESCRIPTION
**Pull request by martndemus**

### CODE REVIEW

File: addon/helpers/compute.js [hunk 1]
```diff
@@ -0,0 +1,7 @@
+import Ember from 'ember';
+
+export function compute([action, ...params]) {
+  return action(...params);
+}
+
+export default Ember.Helper.helper(compute);
```
Comment (poteto) [inline addon/helpers/compute.js]: Small nitpick: Can we destructure the `helper`?

File: README.md [hunk 2]
```diff
@@ -26,6 +26,13 @@ Pipes the return values of actions in a sequence of actions.
 </button>
 ```
 
+### Compute
+Calls an action as an template helper.
+
+```hbs
+The square of 4 is {{compute (action 'square') 4}}
```
Comment (poteto) [inline README.md]: Double quotes please

File: tests/integration/helpers/compute-test.js [hunk 3]
```diff
@@ -0,0 +1,13 @@
+import { moduleForComponent, test } from 'ember-qunit';
+import hbs from 'htmlbars-inline-precompile';
+
+moduleForComponent('compute', 'Integration | Helper | {{compute}}', {
+  integration: true
+});
+
+test('It calls an action and returns it\'s value', function(assert) {
```
Comment (poteto) [inline tests/integration/helpers/compute-test.js]: Prefer double quotes when you need to use `'`

File: tests/integration/helpers/compute-test.js [hunk 4]
```diff
@@ -0,0 +1,13 @@
...
+
+moduleForComponent('compute', 'Integration | Helper | {{compute}}', {
+  integration: true
+});
+
+test("It calls an action and returns it's value", function(assert) {
+  this.on('square', (x) => x * x);
+  this.render(hbs`{{compute (action "square") 4}}`);
+
+  assert.equal(this.$().text().trim(), '16');
```
Comment (poteto) [inline tests/integration/helpers/compute-test.js]: Assertion message please 😬

File: app/helpers/compute.js [hunk 5]
```diff
@@ -0,0 +1 @@
+export { default, compute } from 'ember-functional-helpers/helpers/compute';
```
Comment (poteto) [inline app/helpers/compute.js]: Can we also export in https://github.com/DockYard/ember-functional-helpers/blob/remove-compose/addon/index.js please

**Comment by martndemus** [comment]
@homu r+

**Pull request merged**